packages/loot-core/src/client/data-hooks/accounts.tsx

```diff
@@ -11,7 +12,7 @@ function useAccounts() {
 let AccountsContext = createContext(null);
 
 export function AccountsProvider({ children }) {
-  let data = useAccounts();
+  const data: AccountEntity[] = useAccounts();
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏can you move this type to `useAccounts` hook instead?
```
function useAccounts(): AccountEntity[] {
```